src/script/components/TextInput/TextInput.tsx

```diff
@@ -96,7 +98,9 @@ const TextInput: React.ForwardRefRenderFunction<HTMLDivElement, UserInputProps>
         </span>
       )}
 
+      {/* eslint-disable */}
       <input
+        autoFocus={autofocus}
```

Comment:
I think you should use a `ref` + `useEffect` to trigger the focus at first render. Can you try playing with this?